src/extensions/surveys.ts

```diff
@@ -635,7 +635,7 @@ export const createMultipleChoicePopup = (posthog: PostHog, survey: Survey, ques
                 : formElement.querySelectorAll('input[type=checkbox]:checked')
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-ignore // TODO: Fix this, error because it doesn't recognize node list as an array
-        if (selectedChoices && (selectedChoices.length ?? 0) > 0) {
+        if ((singleOrMultiSelect === 'single_choice' && selectedChoices) || (selectedChoices.length ?? 0) > 0) {
```

Comment:
any reason not to use querySelectorAll above for single choice as well?

Reply:
THAT MAKES A LOT OF SENSE

Reply:
wait i'm going to test it real quick locally first though

Reply:
ok it works